app/models/mainstream_browse_page.rb

```diff
@@ -19,12 +19,16 @@ def active_top_level_browse_page
     linked_items("active_top_level_browse_page").first
   end
 
+  def ordered_second_level_browse_pages
+    details["ordered_second_level_browse_pages"]
+  end
+
   def second_level_browse_pages
     links = linked_items("second_level_browse_pages")
 
-    if second_level_pages_curated?
+    if second_level_pages_curated? and ordered_second_level_browse_pages
```

Comment:
The worry it's probably unnecessary given that collections-publisher is the only app able to publish mainstream browse pages. Might remove this commit after product review.